plugins/mumble_positional_audio_utils.h

```diff
@@ -45,6 +45,32 @@ static inline std::wstring utf8ToUtf16(const std::string &str) {
 	}
 }
 
+// https://social.msdn.microsoft.com/Forums/en-US/8f40dcd8-c67f-4eba-9134-a19b9178e481/vs-2015-rc-linker-stdcodecvt-error
+#if _MSC_VER >= 1900
```

Comment:
Working around a compiler bug also falls into the category of "it clutters the code" while it works around a bug in another application.

I would tend to not do that...